plugins/keepkey/plugin.py

```diff
@@ -138,7 +138,7 @@ def get_client(self, keystore, force_pair=True):
         return client
 
     def get_coin_name(self):
-        return "Testnet" if constants.net.TESTNET else "Bitcoin Gold"
+        return "Testnet" if constants.net.TESTNET else "BitcoinGold"
```

Comment:
If it follows Trezor's code, it should be:
https://github.com/BTCGPU/electrum/blob/9ccaacedfe8f0fe378348a16ff19292bd3c05808/plugins/trezor/trezor.py#L177

Reply:
Nvm. They are using different coin strings.